test/e2e/lib/browser-manager.js

```diff
@@ -0,0 +1,119 @@
+/**
```

Comment:
Should we add a note in the docs or maybe the filename that this is for playwright and not for legacy tests?

Reply:
Sure, I can do that. Will be coming in next commit.